ams/server/src/main/java/com/netease/arctic/server/dashboard/DashboardServer.java

```diff
@@ -394,8 +394,8 @@ public void preHandleRequest(Context ctx) {
           ctx.queryParam("signature"),
           ctx.queryParamMap());
     } else if (needLoginCheck(uriPath)) {
+      LOG.error("---+++++: " + ctx.path());
```

Comment:
Is this redundant? It will introduce too many error logs.

Reply:
> Is this redundant? It will introduce too many error logs.

Oh, it's really my fault. Thanks for your comment. I have removed the unused code, plz check it again.